streamingpro-core/src/main/java/tech/mlsql/crawler/udf/Functions.scala

```diff
@@ -100,3 +148,107 @@ object Functions {
   }
 
 }
+
+class FunctionsUtils(override val uid: String) extends DslTool with  WowParams with Logging {
```

Comment:
Should we remove the DslTool/WowParams trait here?  and the uid is not needed also.